test/src/tests-store.ts

```diff
@@ -254,6 +254,422 @@ describe('Store', () => {
         });
     });
 
+    describe('#getConsentState', () => {
+        it('should return a consent state object from the store', () => {
+            const store: IStore = new Store(
+                sampleConfig,
+                window.mParticle.getInstance()
+            );
+
+            store.persistenceData[testMPID] = {
+                con: {
+                    gdpr: {
+                        analytics: {
+                            c: true,
+                            d: 'foo gdpr document',
+                            h: 'foo gdpr hardware id',
+                            l: 'foo gdpr location',
+                            ts: 10,
+                        },
+                    },
+                    ccpa: {
+                        data_sale_opt_out: {
+                            c: false,
+                            d: 'foo ccpa document',
+                            h: 'foo ccpa hardware id',
+                            l: 'foo ccpa location',
+                            ts: 42,
+                        },
+                    },
+                },
+            };
```

Comment:
I agree that we should move the sample consent state object into a variable so it can be reused, but we can't use it to compare the output of getGDPRConsentState() since inputs and outputs don't technically match. We store a minified object in persistence and the output is the "expanded" transformation.